pkg/kstatus/status/core.go

```diff
@@ -74,6 +75,58 @@ func alwaysReady(u *unstructured.Unstructured) (*Result, error) {
 	}, nil
 }
 
+func hpaConditions(u *unstructured.Unstructured) (*Result, error) {
```

Comment:
Can you add a comment that describes what this is expecting for HPA to be detected as Current?